runtime/ast/precedence.go

```diff
@@ -91,4 +91,7 @@ const (
 	// - FunctionExpression
 	// - PathExpression
 	precedenceLiteral
+	// precendenceAttach is the precedence of
+	// - AttachExpression
+	precendenceAttach
```

Comment:
Is this used by follow-up PRs?

Why do we need another precedence level, and one that binds tighter than literals?

Reply:
I guess I am not completely sure we do, I just wasn't sure what the precedence of `attach` should be, since it is a form of expression than any of the ones that already exist; it's "binary" in its arity but it's not infix like the other binary operators. I.e. the form is `<KEYWORD> <EXPR> <KEYWORD> <EXPR>` instead of `<EXPR> <KEYWORD> <EXPR>`. If we can reuse an existing precedence for this though that would probably be better, just wasn't sure which it should be.

Reply:
After a review session, determined this can probably be `precedenceLiteral`